packages/nextjs/components/scaffold-stark/Faucet.tsx

```diff
@@ -89,9 +89,11 @@ export const Faucet = () => {
       setLoading(false);
       setInputAddress(undefined);
       setSendValue("");
+      notification.success("ETH sent successfully!");
```

Comment:
i am always getting a success message even if i input an invalid `address`, the network request fails but the success message persists.

Reply:
Alright, I'll look into it